spec/forms/change_case_summary_form_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe ChangeCaseSummaryForm do
+  subject(:form) { described_class.new(summary: summary) }
+
+  let(:summary) { "New summary" }
+
+  describe "#valid?" do
+    it "is valid" do
+      expect(form).to be_valid
+    end
+
+    it "does not contain error messages" do
+      form.validate
+      expect(form.errors.full_messages).to be_empty
+    end
+
+    shared_examples_for "invalid form" do |*errors|
+      it "is not valid" do
+        expect(form).to be_invalid
+      end
+
+      it "populates an error message" do
+        form.validate
+        errors.each do |property, message|
+          expect(form.errors.full_messages_for(property)).to eq([message])
+        end
+      end
+    end
+
+    context "when no summary is supplied" do
+      let(:summary) { nil }
```

Comment:
Might be more realistic test a whitespace-only string?

```suggestion
      let(:summary) { " " }
```